doc/release-notes/5.11-release-notes.md

```diff
@@ -0,0 +1,195 @@
+# Dataverse Software 5.11
+
+This release brings new features, enhancements, and bug fixes to the Dataverse Software. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights
+
+### Terms of Access or Request Access Required for Restricted Files
+
+Beginning in this release, Restricted Files must have either Terms of Access or Request Access enabled. This change is to ensure that for each file in a Dataverse installation there is a clear path to get to the data, either through requesting access to the data or to provide context about why requesting access is not enabled.
```

Comment:
One final suggestion - can we say here "Datasets with Restricted Files"? ( to match what is in the warning banner and since the terms or Request Access are at the dataset level)

Reply:
Good idea. Thanks! Fixed in 85d400f3ec.